docs/src/content/docs/basic-concepts.mdx

```diff
@@ -60,19 +80,19 @@ GenSX uses JSX to compose components into workflows. Data flows from parent to c
 </ParentComponent>
 ```
 
-Unlike React's concurrent rendering model, GenSX components execute sequentially from top to bottom. Each component completes before its children begin.
+Unlike React's concurrent rendering model, GenSX evaluates your workflow as a dependency graph. Components execute in parallel whenever possible, while automatically ensuring that all required inputs are available before a component starts executing.
```

Comment:
Do you think we need to talk about trees vs graphs at all here? https://gensx.dev/why-jsx/#graphs-dags-and-trees

Maybe something like: 

Just because GenSX uses trees doesn't mean it can't represent cycles and everything else you would expect from graphs and agents. To learn more see (https://gensx.dev/why-jsx/#graphs-dags-and-trees)...

Reply:
ooh yeah i like that; will add something in

Reply:
Here's what i landed on. Not perfect but workable.

---

While GenSX uses a [tree-based structure with JSX](https://gensx.dev/why-jsx/#graphs-dags-and-trees), it can represent everything you'd expect from graph-based workflows including cycles and agentic patterns